main.js

```diff
@@ -1,6 +1,48 @@
-var studyButtons = document.querySelector(".activity-buttons");
+var activityButtons = document.querySelector(".activity-buttons");
+var startActivityBtn = document.querySelector(".start-activity-button");
+var timeInput = document.querySelector(".time-input");
+var currentActivity;
+
+startActivityBtn.addEventListener('click', startActivity);
+activityButtons.addEventListener("click", selectActivity);
+timeInput.addEventListener("keydown", noLetters);
+
+// Start HERE next time
 
-studyButtons.addEventListener("click", selectActivity);
+function startActivity(event) {
+  makeNewActivity(event);
+  toggleElement("new-activity-title", "new-activity-form");
+  toggleElement("current-activity-title", "timer-display");
+  // if statement to assign color to start button circle
+}
+
+function toggleElement(className1, className2) {
+  document.querySelector(`.${className1}`).classList.toggle(`hidden`);
+  document.querySelector(`.${className2}`).classList.toggle(`hidden`);
+}
```

Comment:
@Abdeboskey should `hidden` be in quotes instead of backticks since it's a class?

Reply:
Great catch Steph. Functionally, I think they are doing the same thing, but for the sake of clarity and consistency we should absolutely use " " instead.